roles/httpd/tasks/main.yml

```diff
@@ -34,6 +34,16 @@
 - name: Restore SELinux Contexts in Document Root
   shell: restorecon -R /var/www/html
 
+- name: Enable webserver ports in firewall
+  firewalld:
+    port: "{{ item }}/tcp"
+    state: enabled
+    permanent: yes
+    immediate: yes
+  loop:
+    - "{{ httpd_port }}"
+    - 22623
```

Comment:
22623 shouldn't be needed on the web server. This is the port for the machine config operator running on the cluster.

Reply:
True, I kind of got confused around that one